examples/http-request-errorbudget.yaml

```diff
@@ -0,0 +1,37 @@
+recordingrule:
+- expr: |
+    sum(label_replace(increase(promhttp_metric_handler_requests_total{namespace="default",job="fooapp"}[30d]), "status_class", "${1}xx", "code", "([0-9])..")) by (status_class)
+  labels:
+    job: fooapp
+    namespace: default
+  record: status_class:promhttp_metric_handler_requests_total:increase30d
```

Comment:
It's more common to strip the `_total` in the name of a rate/increase recording rule.

Reply:
While this is something that I'd like todo and we can do on a best effort basis by using `std.replace(s, '_total', '')` I'm not too sure if this is really want we want, cause then it'll be inconsistent for all non _total use cases :man_shrugging: